images/Dockerfile.ubuntu-20-lts-arm-rpi

```diff
@@ -0,0 +1,61 @@
+ARG BASE_IMAGE=ubuntu:20.04
+
+FROM $BASE_IMAGE
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get update && \
```

Comment:
This is a formatting nit-pick, but in the other ubuntu Docker files, the format is like this (with the `&&` after the newline):
```
RUN apt-get update \
  && ...
```

I don't know if there is a convention, but it'd be nice to be consistent.